modules/loader/src/test/java/com/opengamma/strata/loader/CharSourcesTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2017 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.loader;
+
+import static org.testng.Assert.assertEquals;
+
+import java.io.File;
+import java.nio.file.Paths;
+
+import org.testng.annotations.Test;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.CharSource;
+import com.opengamma.strata.collect.io.ArrayByteSource;
+import com.opengamma.strata.collect.io.ResourceLocator;
+
+public class CharSourcesTest {
```

Comment:
Strata tests have Javadoc

    /**
     * Test {@link CharSources}.
     */

And yes, links are use here.